src/olympia/translations/tasks.py

```diff
@@ -39,3 +41,25 @@ def update_outgoing_url(pks, *, old_outgoing_url, **kw):
             Value(settings.REDIRECT_URL),
         ),
     )
+
+
+@task
+@use_primary_db
+def strip_html_from_summaries(pks, **kwargs):
+    """
+    Run translations of Add-on summaries through bleach to strip them
+    of HTML.
+
+    Used to clean up old summaries from when we accepted URLs in summaries and
+    turned them into HTML.
+    """
+    # Note: can't just use PureTranslation.clean(), because we want to strip
+    # the HTML and not just escape it here - We used to create that HTML
+    # automatically, so it's out responsability to remove it, escaping it not
+    # enough.
+    cleaner = bleach.Cleaner(tags=[], attributes={}, strip=True)
+    translations = Translation.objects.filter(pk__in=pks)
+    for translation in translations:
+        translation.localized_string = cleaner.clean(translation.localized_string)
+        translation.localized_string_clean = translation.localized_string
+        translation.save()
```

Comment:
Do we need to trigger ES reindexing, or is there already a signal connected to the Translation instances?

Reply:
Actually... either way, is there a way to verify that in tests?  ("no" is a valid answer 😬)

Reply:
That's a good point - since I'm updating the translations directly there wouldn't be an automatic reindex, we need to trigger that ourselves. Testing is easy, with a `ESTestCase` and performing a search after we've updated the translations. Implementing it is going to be annoying as we need to do the reverse query to find the add-on from the translation id, but that's doable, I'll look into it.

Reply:
https://github.com/mozilla/addons-server/pull/23119/commits/f240a5da3101fa926325ce8af29d7c2cadf970ee (ended up just mocking `index_addons`, no need for a real ES test case)